pkg/api/components_test.go

```diff
@@ -780,7 +780,6 @@ func TestSynthesizeComponentsConfig(t *testing.T) {
 	for _, c := range cases {
 		c := c
 		t.Run(c.name, func(t *testing.T) {
-			t.Parallel()
```

Comment:
Can these not run in parallel now, or was this for debugging?

Reply:
I have this split up in a separate commit.
Tests are modifying `c.components`, is shared between a couple of cases.

I got a concurrent read/write panic here.